templates/version.ts.hbs

```diff
@@ -0,0 +1,2 @@
+export const USER_AGENT_HEADER = "user-agent";
+export const USER_AGENT_VALUE = "commerce-sdk-isomorphic@{{metadata.sdkVersion}}";
```

Comment:
The node SDK uses the user agent `commerce-sdk@version;`, with the `;` included in the user agent so that it ends on a known character. Not sure that it's really necessary, but that's what it does, for comparison.

Reply:
since we use semvar, we know the the ending chars are numbers and also user agents are tokenized using space that eliminates the need for a specific ending char...